capstone/src/main/java/com/google/sps/servlets/YoutubeTrendingServlet.java

```diff
@@ -36,16 +36,23 @@
   private static final String KEY = "API_KEY";
   private static final String NUM_RESULTS = "2";
   private static final MediaType JSON = MediaType.get("application/json; charset=utf-8");
+  private String postRequestBodyData;
   OkHttpClient client = new OkHttpClient();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String videoCategoryId = request.getParameter("videoCategoryId");
-    String completeUrl = BASE_URL + "&maxResults=" + NUM_RESULTS + "&regionCode=US&videoCategoryId=" + videoCategoryId + "&key=" + KEY;
+    String completeUrl = BASE_URL + "&maxResults=" + NUM_RESULTS + "&regionCode=US&videoCategoryId=" + postRequestBodyData + "&key=" + KEY;
+    System.out.println(completeUrl);
```

Comment:
Let's remove this print statement

Reply:
addressed in "address nits and add youtube caller support"